packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1863,19 +1872,10 @@ abstract class MultiSelectableSelectionContainerDelegate extends SelectionContai
   static int _compareHorizontally(Rect a, Rect b) {
     // a encloses b.
     if (a.left - b.left < precisionErrorTolerance && a.right - b.right > - precisionErrorTolerance) {
-      // b ends before a.
```

Comment:
I added these in https://github.com/flutter/flutter/pull/127478 , but looking at them again I don't think they make sense. If a is encompassed by b then we can assume a is higher since it started before b. If b is encompassed by a then we can assume b is higher since it started before a.